Demo/CarthageExample/CarthageExampleUITests/EarlGrey.swift

```diff
@@ -0,0 +1,177 @@
+//
+// Copyright 2018 Google Inc.
```

Comment:
Can't you link to the internal EG.swift file? That'll help us test it with Bitrise.

Reply:
Great feedback! Done.